src/panels/config/integrations/dialog-add-integration.ts

```diff
@@ -188,6 +189,9 @@ class AddIntegrationDialog extends LitElement {
                   ([dom, val]) => val.name || domainToName(localize, dom)
                 )
               : undefined,
+            domains: integration.integrations
+              ? Object.keys(integration.integrations || {})
```

Comment:
```suggestion
              ? Object.keys(integration.integrations)
```
You already checked its existence on the line above.